docs/products/kafka/howto/snowflake-open-catalog.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Configure the Iceberg sink connector with Snowflake Open Catalog
+sidebar_label: Snowflake Open Catalog (Polaris)
+---
+
+import CreateIcebergSinkConnector from "@site/static/includes/create-iceberg-sink-connector.md";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from '@site/src/components/ConsoleIcons';
+import RelatedPages from "@site/src/components/RelatedPages";
+
+[Snowflake Open Catalog](https://other-docs.snowflake.com/en/opencatalog/overview) is a managed [Apache Polaris™](https://polaris.apache.org/) service that supports the [Apache Iceberg™](https://iceberg.apache.org/) REST catalog API and uses Amazon S3 for storage.
+
+Use the Iceberg sink connector in Aiven for Apache Kafka® Connect to write data to
+Iceberg tables managed by Snowflake Open Catalog.
+
+## Prerequisites
+
+- A [Snowflake Open Catalog account](https://other-docs.snowflake.com/en/opencatalog/create-open-catalog-account)
+- A Polaris catalog created using
+  [Amazon S3 as the storage backend](https://other-docs.snowflake.com/en/opencatalog/create-catalog#create-a-catalog-using-amazon-simple-storage-service-amazon-s3)
+- An [Aiven for Apache Kafka® service](/docs/products/kafka/kafka-connect/howto/enable-connect)
+  with Apache Kafka Connect enabled, or
+  a [dedicated Aiven for Apache Kafka Connect® service](/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster)
+- An Apache Kafka topic that contains data to sink
+- An Apache Kafka control topic (default: `control-iceberg`)
+- AWS S3 credentials and a configured S3 bucket
+- Polaris authentication credentials
+  - `iceberg.catalog.scope`: A role scope defined in the Open Catalog
+  - `iceberg.catalog.credential`: A token or hash for authenticating with the catalog
+
+:::note
+Snowflake Open Catalog does not support automatic catalog creation. You must create the
+catalog manually in the Snowflake console.
+:::
+
+## Create an Iceberg sink connector configuration
+
+To configure the Iceberg sink connector, define a JSON configuration with the required catalog, S3, and Kafka settings.
+
+:::note
+Loading worker properties is not supported. Use `iceberg.kafka.*` properties instead.
+:::
+
+```json
+{
+  "name": "<your-connector-name>",
```

Comment:
This comment refers to all placeholder instances on this PR:
Normally, we use ALL-CAPS for placeholders in the Aiven docs. Please considers using the following for consistency:

```suggestion
  "name": "CONNECTOR_NAME",
```
and relevant alternatives for all the other placeholders.